bin/pyenv-offline-installer

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+#
+# Usage:
+#   pyenv-offline-installer [<path-to-archive>]
+# 
+# if <path-to-archive> is omitted, the script would search for
+# "pyenv-package.tar.gz" in current working directory.
+
+set -e
+[ -n "$PYENV_DEBUG" ] && set -x
+
+if [ -z "$PYENV_ROOT" ]; then
+  PYENV_ROOT="${HOME}/.pyenv"
+fi
+
+shell="$1"
+if [ -z "$shell" ]; then
+  shell="$(ps c -p "$PPID" -o 'ucomm=' 2>/dev/null || true)"
+  shell="${shell##-}"
+  shell="${shell%% *}"
+  shell="$(basename "${shell:-$SHELL}")"
+fi
+
+colorize() {
+  if [ -t 1 ]; then printf "\e[%sm%s\e[m" "$1" "$2"
+  else echo -n "$2"
+  fi
+}
+
+conditional_mv() {
+  [ -d "$2" ] || mkdir -p "$2" && mv "$1"/* "$2"
```

Comment:
Could you rewrite this with using if then fi? I think it'd be better to clarify the operator precedence.